scripts/join-release-testnet.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+# Set up a Exocore service to join the Exocore Release testnet.
+
+# Load environment variables from .env file
+if [ -f .env ]; then
+  export $(cat .env | xargs)
+fi
```

Comment:
I think we can remove this because the user should call `source .env` before running this file.